diff --git a/mcomix/run.py b/mcomix/run.py
index ea93bae..0e2b40a 100755
--- a/mcomix/run.py
+++ b/mcomix/run.py
@@ -124,7 +124,7 @@ def run():
         require_version('Gtk', '3.0')
         require_version('Gdk', '3.0')
 
-        from gi.repository import Gdk, Gtk, GObject
+        from gi.repository import Gdk, Gtk, GObject, GLib
 
         GObject.threads_init()
 
@@ -178,6 +178,7 @@ def run():
         Gtk.widget_set_default_direction(Gtk.TextDirection.RTL)
 
     Gdk.set_program_class(constants.APPNAME)
+    GLib.set_prgname("net.sourceforge.mcomix")
 
     settings = Gtk.Settings.get_default()
     # Enable icons for menu items.
